change/@internal-react-composites-3e8203bf-9f73-4300-8ae4-34b282a1e57a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Created new Notification Icon for the chat button in the meeting composite.",
```

Comment:
This changelog is for end-users.
So a tip is to avoid using terms like `created` and use `Add` or `Introduce` instead. ;)
Even `Notification Icon for the chat button in the meeting composite.` this looks good to me if I read it as an end user.

Reply:
okay will update that!